pyproject.toml

```diff
@@ -36,7 +36,9 @@ keywords = ["tracking","mot","sort","deepsort", "machine-learning", "deep-learni
 dependencies = [
     "firerequests>=0.1.2",
     "numpy>=2.0.2",
+    "safetensors>=0.5.3",
     "supervision>=0.26.0rc7",
+    "tensorboard>=2.19.0",
     "validators>=0.34.0",
```

Comment:
Is this temporary and permanent ? because based on code the you also need to include torch too.

Reply:
I think we should create a separate extra called `reid` and move these.